UnityProject/Assets/Scripts/UI/Canister/NumberSpinner.cs

```diff
@@ -115,11 +127,13 @@ public void DisplaySpinTo(int newValue)
 
 		return;
 
+		//NOTE: Previously tried to implement a spinning animation, but now am bypassing that stuff because it was
+		//proving difficult to make it look good but also be responsive when pressure is changing rapidly.
+		//Currently it just jumps directly to the number.
```

Comment:
Yeah there is a point where rotations on screen are too fast to look good.
Having them too-slow makes it hard to read, so in that case: Functionality  before looks is okey!